frontend/src/components/Pages/FrontPage/MissionOverview/MissionQueueView.tsx

```diff
@@ -132,22 +132,27 @@ export function MissionQueueView({ refreshInterval }: RefreshProps) {
         })
     }, [missionQueue, ongoingMissions])
 
-    const missionQueueDisplay = missionQueue.map(function (mission, index) {
-        return <MissionQueueCard key={index} mission={mission} onDeleteMission={onDeleteMission} />
-    })
-
-    const loadingQueueDisplay = () => {
-        return <MissionQueueCard key={'test'} mission={dummyMission} onDeleteMission={() => {}} />
-    }
+    const missionQueueDisplay = missionQueue.map((mission, index) => (
+        <MissionQueueCard key={index} order={index + 1} mission={mission} onDeleteMission={onDeleteMission} />
+    ))
+
+    const loadingQueueDisplay = (
+        <MissionQueueCard
+            key={'test'}
```

Comment:
instead of test for the key I would use placeholder or something, i know this is not your code but maybe you do the change?